apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/backend/ast/Ast.kt

```diff
@@ -1,5 +1,8 @@
 package com.apollographql.apollo.compiler.backend.ast
 
+import com.squareup.kotlinpoet.MemberName
+import com.squareup.kotlinpoet.TypeName
+
```

Comment:
Idea was that this class AST supposed to be free from any specific kotlin / java poet classes and can be used with any code generation libraries. With these changes we lock AST to be used only with Kotlin Poet.

Reply:
Good catch! I'll fix this.